crossword-xml-uploader/src/main/scala/com/gu/crossword/crosswords/CrosswordStore.scala

```diff
@@ -40,4 +40,9 @@ trait CrosswordStore {
     s3Client.deleteObject(config.crosswordsBucketName, awsKey)
   }
 
+  def archiveFailedCrosswordXMLFile(config: Config, awsKey: String): Unit = {
+    val processingFailedBucketName = "crossword-failed-files"
+    s3Client.copyObject(config.crosswordsBucketName, awsKey, processingFailedBucketName, awsKey)
+    s3Client.deleteObject(config.crosswordsBucketName, awsKey)
```

Comment:
worth checking the copy succeeded above before deleting?

Reply:
https://github.com/guardian/crossword-uploader/pull/31/commits/2d66985aa19c0c9788f84179a4fcd9fc3eb42416

Have tested this in CODE and still works as expected